src/domain/child/ChildUtils.ts

```diff
@@ -38,3 +39,15 @@ export const getTranslatedRelationshipOptions = (
     },
   ];
 };
+
+/**
+ * Child data which was submit from form supposed to have same
```

Comment:
I think this sentence is a bit clearer than the comment: 

"The child data submitted from the form is supposed to have the same types as the child data received from backend"

Reply:
make sense 👍